website/blog/2021-03-05-GAAD-React-Native-Accessibility.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The GAAD Pledge - Improving React Native Accessibility
+author: Alexandra Marlette
+authorTitle: GAAD Pledge Open Source Accessibility Community Manager for React Native
+authorURL: 'https://twitter.com/alexmarlette'
+authorImageURL: 'https://avatars.githubusercontent.com/u/10052470?s=460&u=7f2304cb929d1de703856717af86324c66728f3a&v=4'
+authorTwitter: alexmarlette
+tags: [announcement]
+---
+
+## Hello React Native Community,
+
+In May 2020 Facebook was the first company to take the [GAAD pledge](https://diamond.la/GAADPledge/), by doing so they committed to making accessibility a core part of the React Native open source project. Since May, Facebook has spent that time thoughtfully reviewing and documenting accessibility gaps within React Native. So far the gap analysis has surfaced 90 issues, all of which have been translated to Github issues on [React Native's Github](https://github.com/facebook/react-native/projects/15).
```

Comment:
Since this is being published on the React Native blog, you can probably change 

```
translated to Github issues on [React Native's Github](https://github.com/facebook/react-native/projects/15).
```

to 
```
translated to [Github issues](https://github.com/facebook/react-native/projects/15).
```